_literate/A-model-tuning.jl

```diff
@@ -31,10 +31,10 @@ r   = range(dtc, :max_depth, lower=1, upper=5)
 
 tm = TunedModel(model=dtc, ranges=[r, ], measure=cross_entropy)
 
-# Note that "wrapping a model in a tuning strategy" as above means creating a new "self-tuning" version of the model, tuned_model = TunedModel(model=...), in which further key-word arguments specify:
-# 1. the algorithm (a.k.a., tuning strategy) for searching the hyper-parameter space of the model (e.g., tuning=Random(rng=123)or tuning=Grid(goal=100)).
-# 2. the resampling strategy, used to evaluate performance for each value of the hyper-parameters (e.g., resampling=CV(nfolds=9, rng=123) or resampling=Holdout(fraction_train=0.7)).
-# 3. the measure (or measures) on which to base performance evaluations (and for reporting purposes) (e.g., measure = rms or measures=[rms, mae]).
+# Note that "wrapping a model in a tuning strategy" as above means creating a new "self-tuning" version of the model, `tuned_model = TunedModel(model=...)`, in which further key-word arguments specify:
+# 1. the algorithm (a.k.a., tuning strategy) for searching the hyper-parameter space of the model (e.g., `tuning=Random(rng=123)or tuning=Grid(goal=100)`).
```

Comment:
please use two backticks in between (so that the "or" is not in backticks) so 

```
`tuning = Random(rng=123)` or `tuning = Grid(goal=100)`
```

I recommend using hackmd.io to directly see what markdown looks like if you want.

Reply:
This is still a tiny bit confusing (too many cooks, maybe). How about:

Once a range of values has been defined, you can then wrap the model in a `TunedModel`: ~~specifying the tuning strategy.~~

tm = TunedModel(model=dtc, ranges=[r, ], measure=cross_entropy)

~~Note that "wrapping a model in a tuning strategy" as above means~~ This essentially ~~creating~~ creates a new "self-tuning" version of the model, ...